html-css/examples/css/chapter16.css

```diff
@@ -0,0 +1,120 @@
+/* CONTROLLING SIZES OF IMAGES IN CSS */
+img.large {
+    width: 500px;
+    height: 500px;
+}
+
+img.medium {
+    width: 250px;
+    height: 250px;
+}
+
+img.small {
+    width: 100px;
+    height: 100px;
+}
+
+
+/* ALIGNING IMAGES USING CSS */
+
+img.align-left {
+    float: left;
+    margin-right: 10px;
+}
+
+img.align-right {
+    float: right;
+    margin-left: 10px;
+}
+
+img.aligning-medium {
+    width: 250px;
+    height: 250px;
+}
+
+
```

Comment:
Remove redundant empty line, one line empty between css selectors group

Reply:
Thank you for your comments. I will learn from experience